sdk/communication/azure-communication-identity/src/main/java/com/azure/communication/identity/CommunicationIdentityAsyncClient.java

```diff
@@ -62,6 +63,7 @@ public final class CommunicationIdentityAsyncClient {
 
     private final CommunicationIdentitiesImpl client;
     private final ClientLogger logger = new ClientLogger(CommunicationIdentityAsyncClient.class);
+    private final CommunicationIdentityClientUtils utils = new CommunicationIdentityClientUtils();
```

Comment:
no need to create an instance, as the methods would be static and you can access directly through class, same for the sync client class

Reply:
Removed. Methods are now static.